lib/browser_collection.js

```diff
@@ -1,49 +1,57 @@
+'use strict'
+
 var EXECUTING = require('./browser').STATE_EXECUTING
 var Result = require('./browser_result')
 
-var Collection = function (emitter, browsers) {
-  browsers = browsers || []
-
-  this.add = function (browser) {
-    browsers.push(browser)
-    emitter.emit('browsers_change', this)
+class BrowserCollection {
+  /**
+   * @param {EventEmitter} emitter the emitter to send events to
+   * @param {Array} browsers the array of browsers to wrap
+   */
+  constructor (emitter, browsers) {
+    this.browsers = browsers || []
+    this.emitter = emitter
+  }
+  add (browser) {
```

Comment:
Personally I think it would be nice when this method had a similar api like `Array.prototype.push`, I mean adding multiple items, like - `collection.add(new Browser(), new Browser(), ..., new Browser())`